man/os-release.xml

```diff
@@ -61,7 +61,8 @@
     shell style. All strings should be in UTF-8 format, and
     non-printable characters should not be used. It is not supported
     to concatenate multiple individually quoted strings. Lines
-    beginning with "#" shall be ignored as comments.</para>
+    beginning with "#" shall be ignored as comments. Blank lines are
+    allowed.</para>
```

Comment:
should really say what they are used for, i.e. nothing. maybe: "… are permitted and ignored." or so...

Reply:
Good idea. Done.